### ISSUE TITLE
Folders' view in workspace

### ISSUE THREAD

**Issue by massimoazzolini** [ploneintranet/ploneintranet#435]
in http://localhost:8080/ploneintranet/workspaces/open-market-committee/manage-information/ you see the folder's header and a grey area

![image](https://cloud.githubusercontent.com/assets/883251/8574199/c81a3e2e-2595-11e5-9eec-52c2d83489eb.png)

**Comment by pilz** [comment]
We actually don't have a folder view at the moment. I am not sure whether we want one, the folder is represented in the sidebar. However then we must avoid that users get to a folder view. E.g. Folders should then not be included in search results etc.

**Comment by massimoazzolini** [comment]
I agree for now. this is definitely minor.
Anyway, at a some point we'll need to manage it: users can always enter the url of a folder.
What about to show the expanded sidebar (a sort of replacement for the usual plone folder view)? or, maybe easier to implement, a white (not grey) area?

**Comment by pilz** [comment]
Exactly. We must definitely find a solution for the current state. I flagged it as UX Question so that we can get also the Design input. Then we fix, one way or the other :)

**Comment by gyst** [comment]
Related to #524 and #386
